packages/core/src/features/authSlice.js

```diff
@@ -31,6 +32,7 @@ export const login = createAsyncThunk(
       password,
     );
     const userId = await getActiveUserId();
+    storage.setItem('url', urlWithProtocol);
```

Comment:
A better practice is to give meaning to keys, just 'url' does not really explain what it is used for. Also, it could be better to define this key only one time with an exported variable and use this variable in login screen:

```
   const URL_STORAGE_KEY = 'LastConnectionUrl';
...
    storage.setItem(URL_STORAGE_KEY, urlWithProtocol);
```